util.js

```diff
@@ -52,3 +52,36 @@ exports.getTestModifiers = function getTestModifiers(node) {
 
 	return [];
 };
+
+const getMembers = node => {
+	const name = node.property.name;
+
+	if (node.object.type === 'MemberExpression') {
+		return getMembers(node.object).concat(name);
+	}
+
+	return [name];
+};
+
+exports.getMembers = getMembers;
+
+const assertionMethodsNbArguments = {
```

Comment:
Why the preference for Map and Set? Except for holding objects as keys for Map, I don't know the benefits. Don't mind changing it, as I don't see drawbacks to it, mostly curious.